app/main/views/brief_responses.py

```diff
@@ -230,21 +237,13 @@ def list_brief_responses():
         )
 
     if brief_id or supplier_id:
-        return jsonify(
-            briefResponses=[brief_response.serialize() for brief_response in brief_responses.all()],
-            links={'self': url_for('.list_brief_responses', supplier_id=supplier_id, brief_id=brief_id)}
-        )
+        return list_result_response(RESOURCE_NAME, brief_responses)
```

Comment:
Hmm, in the change above you've explicitly added a return code of 200. Do we want to do this explicitly everywhere? If so, you're missing a few places (eg here and below, possibly more).